cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -1115,6 +1115,18 @@ if (jemalloc)
   endif()
 endif()
 
+#---Check for liburing----------------------------------------------------------------
+if (liburing)
+  if(NOT CMAKE_SYSTEM_NAME MATCHES Linux)
+    message(FATAL_ERROR "liburing is only available on Linux")
```

Comment:
Can you silently disable option, and print a message that "liburing was disabled because it is available only on Linux"?

Reply:
Thank you, done in 6f1efa8967efe0071847f2d5abe0681219f7291f